ricecooker/config.py

```diff
@@ -14,13 +14,78 @@
 SUSHI_BAR_CLIENT = None
 STAGE = False
 
-# Setup logger
+# Don't use this - call logging.getLogger(__name__) from each
+# individual module. Logging is configured centrally by calling
+# setup_logging()
 LOGGER = logging.getLogger()
 
-# Setup temporary stream handler (used until full setup in chef.config_logger)
-temporary_log_handler = logging.StreamHandler()
-LOGGER.setLevel(logging.INFO)
-LOGGER.addHandler(temporary_log_handler)
+
+# Keep error log when setup_logging is called
+_ERROR_LOG = None
+
+
+def setup_logging(level=logging.INFO, error_log=None):
+    """
+    Set up logging, useful to call from your sushi chef main script
+    """
+    global _ERROR_LOG
+
+    if not error_log:
+        error_log = _ERROR_LOG
+    else:
+        _ERROR_LOG = error_log
+
+    handlers = ["console"]
+    if error_log:
+        handlers.append("error")
+
+    logging.config.dictConfig({
+        'version': 1,
+        'formatters': {
+            'colored': {
+                '()': 'colorlog.ColoredFormatter',
+                'format': "%(log_color)s%(levelname)-8s%(reset)s %(blue)s%(message)s"
```

Comment:
:+1: 

I think that's a matter of taste, but it's indeed configurable, so I'll add an option!

Before I couldn't tell if things were logged or with `print()`.

Stating the log level by default is necessary for someone who hasn't seen it before to get the idea intuitively and then be like "how can I shorten that", as more experienced cheffers :)

Reply:
Are you okay that the log level is on by default?

Reply:
Can you make it off by default to preserve the current behaviour? (i.e. in matters of taste err on the side of no-change)

Reply:
> Before I couldn't tell if things were logged or with print().
>

With logs being colored now, we'll be able to tell remaining `print` statements since they will be white.

Reply:
I prefer the one that displays the level name - the color coding helps to navigate visually, but the DEBUG/INFO/WARNING/ERROR/CRITICAL stuff has to be written for...

1) a first-time user reading outputs, you would have to read the implementation to understand where the colors come from
2) someone being displayed log-outputs who isn't a developer of the chef or ricecooker (but for whom the log is still useful to look for errors in)
3) a copy-paste to a medium without colors

I think these are reasons why displaying the level name was the default example that I copied in :)

![image](https://user-images.githubusercontent.com/374612/79007588-c07ed780-7b5b-11ea-88e8-b2bae166bd5d.png)